Translation/Extractor/File/FormExtractor.php

```diff
@@ -138,7 +142,7 @@ public function enterNode(Node $node)
                         $this->parseItem($item, $domain);
                         break;
                     case 'choices':
-                        if ($this->parseChoiceNode($item, $node, $domain)) {
+                        if ($this->parseChoiceNode($item, $node, $choiceDomain)) {
                             continue 2;
```

Comment:
Just out of curiosity - why not place the $choiceDomain detection within the parseChoiceNode function? Since it only applies there? Or have the getDomain() function check the choice domain and return that?